include/manager.hpp

```diff
@@ -43,67 +43,47 @@ class Manager
     /**
      * @brief Update keyword value.
      *
-     * API can be used to update VPD keyword.
+     * This API is used to update keyword value on the given i_vpdPath and its
```

Comment:
nit: instead of i_vpdPath say input path

Reply:
done